pkg/cluster/rook_ceph.go

```diff
@@ -233,14 +233,115 @@ func (c *Controller) ReconcileMonCount(count int) error {
 
 	c.Log.Infof("Changing mon count from %d to %d", cluster.Spec.Mon.Count, count)
 	cluster.Spec.Mon.Count = count
-	_, err = c.Config.CephV1.CephClusters("rook-ceph").Update(context.TODO(), cluster, metav1.UpdateOptions{})
```

Comment:
Are we planning to use `context` in some meaningful way? If not, I'd rather use `context.TODO()` where required only.

Reply:
I moved the context.TODO() call up the stack. I'm punting on making a decision what context to use in the call to this function. It is my intention to eventually use context from the process.